Maestro/subscriptions.json

```diff
@@ -78,9 +78,9 @@
             "-Project='tests\\build.proj'",
             "--",
             "/t:UpdateDependenciesAndSubmitPullRequest",
-            "/p:GitHubUser=dotnet-bot",
-            "/p:GitHubEmail=dotnet-bot@microsoft.com",
-            "/p:GitHubAuthToken=`$(`$Secrets['DotNetBotGitHubPassword'])",
+            "/p:GitHubUser=dotnet-maestro-bot",
```

Comment:
dotnet-maestro-bot will need a fork already created for each repo--it doesn't create those itself, only the branches.